enterprise/server/scheduling/task_router/task_router.go

```diff
@@ -61,9 +64,11 @@ func New(env environment.Env) (interfaces.TaskRouter, error) {
 	if rdb == nil {
 		return nil, status.FailedPreconditionError("Redis is required for task router")
 	}
+	strategies := []routingStrategy{runnerRecyclingRoutingStrategy{}, outputAffinityRoutingStrategy{}}
```

Comment:
tbd if we need multiple of these, but maybe it's nice so we can have one for workflows and one for other actions?

Reply:
Per discussion, my thinking was that we'll probably want to at least experiment with other routing strategies in the future, so clarifying this logic is worth a shot.